File: platform/src/main/java/ru/abondin/hreasy/platform/service/dto/VacationDto.java
package ru.abondin.hreasy.platform.service.dto;

import lombok.Data;

import java.time.LocalDate;

@Data
public class VacationDto {
    private int id;
    private int employee;
    private String employeeDisplayName;
    private Integer employeeCurrentProject;
    private int year;
    private LocalDate startDate;
    private LocalDate endDate;
    private String notes;
}

File: platform/src/main/kotlin/ru/abondin/hreasy/platform/api/model/VacationDto.kt
package ru.abondin.hreasy.platform.api.model

import java.time.LocalDate

data class VacationDto (
        val id: Int,
        val employee: Int,
        val emplyeeDisplayName: String,
        var year: Int,
        var startDate: LocalDate?,
        var endDate: LocalDate?,
        var notes: String?
)
